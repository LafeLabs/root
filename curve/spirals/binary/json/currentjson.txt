{
    "constants": {
        "phi": 1.618
    },
    "imgurl": "../symbol/svg/svg1531522267.svg",
    "plotparams": {
        "G": 3731.9110786231904,
        "g": 0.017985878990921366,
        "duration": 545.4545454545454,
        "plotwidth": 400,
        "plotheight": 400,
        "x0": 305.01437587440824,
        "y0": 149.4871798375907
    },
    "funcparams": {
        "T": 1,
        "X0": 0,
        "Y0": 0
    }
}