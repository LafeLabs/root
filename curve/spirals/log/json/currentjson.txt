{
    "constants": {},
    "imgurl": "",
    "plotparams": {
        "G": 611.2716867218143,
        "g": 1.798587899092136,
        "duration": 410.377496908862,
        "plotwidth": 400,
        "plotheight": 400,
        "x0": 204.22402,
        "y0": 207.0463121396247
    },
    "funcparams": {
        "tau": 46.65073802097332,
        "T": 100,
        "X0": 0,
        "Y0": 0
    }
}