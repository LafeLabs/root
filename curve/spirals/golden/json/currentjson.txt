{
    "constants": {
        "phi": 1.618
    },
    "imgurl": "../symbol/geometry/goldenspiral/svg/svg1531179864.svg",
    "plotparams": {
        "G": 1585.4813285073162,
        "g": 0.017985878990921366,
        "duration": 545.4545454545454,
        "plotwidth": 400,
        "plotheight": 400,
        "x0": 271.82217062000007,
        "y0": 72.56846931505859
    },
    "funcparams": {
        "T": -1,
        "X0": 0,
        "Y0": 0
    }
}